toolchain/docker.go

```diff
@@ -15,6 +72,11 @@ func newDockerClient() (*docker.Client, error) {
 		dockerEndpoint = "unix:///var/run/docker.sock"
 	} else if !strings.HasPrefix(dockerEndpoint, "http") && !strings.HasPrefix(dockerEndpoint, "tcp") {
 		dockerEndpoint = "http://" + dockerEndpoint
+	} else if strings.HasPrefix(dockerEndpoint, "tcp") {
+		certPath := os.Getenv("DOCKER_CERT_PATH")
+		if certPath != "" {
```

Comment:
`if certPath := os.Getenv("DOCKER_CERT_PATH"); certPath != "" {` is more concise.